### ISSUE TITLE
Multi-Tabbed PuTTY

### ISSUE THREAD

**Issue by iggypops** [siwells/teaching_set09103#11]
Not an issue really, more of a tip.

Perhaps this isn't news to most of you, but if like me you have found yourself struggling to keep up with all your opened PuTTY sessions, [this free tool](http://ttyplus.com/multi-tabbed-putty/) is a quick and easy solution. Essentially you are still using the same separate PuTTY windows, but nicely wrapped in a clean tabbed GUI, thus no need to learn anything new.

Give me a shout if you need any help.

**Comment by iggypops** [comment]
Can also split the screen and achieve something like this..
![image](https://user-images.githubusercontent.com/32165965/31851714-39e4f552-b663-11e7-8155-854557d8b112.png)

**Comment by siwells** [comment]
Thanks Igors, this looks like it could be an excellent replacement for the default PuTTY tool. How well does it work on the lab machines? Simon

**Comment by iggypops** [comment]
Hi Simon, to be honest didn't really have a chance to test it out on the lab machines yet, but can't imagine any issues, as it is portable and only requires a PuTTY executable to run. Will give it a go later on.
Thanks

**Comment by siwells** [comment]
That would be great. I like to get a few people, including myself, to try out tools and give a recommendation before I introduce new/different tools to the class as getting things that work reliably for 50+ people can be tough. Let me know. If you've got any friends in class who can give it a try, tell them to also comment here. If all goes well, we can add this to the tool list for next week.

**Comment by captainkuromaru** [comment]
I am late to the party, but if you just need to have different context for your work in the terminal (especially now when it's seperate VMs for every student), things like "screen" (it's definitely in fedora repos, but i assume it's also available on ubuntu repo servers) can do the job. it's still gonna be only one window of putty and you can only see one "screen" at the time, but it's easy to browse between tabs and keep your work seperated in contexts (i usually use it to look through logs in one and doing whatever job i am doing in the next one, for example)

**Comment by siwells** [comment]
Yes. I agreen. I personally use screen and both screen and Tmux are mentioned explicitly in the workbook. Both can also be used to split up the window into different workspaces. 

However, there is also an element of balancing the complexity of the work environment. There is a wide range of abilities and backgrounds in our class so having a simple but sub-optimal default, then allowing students with more confidence to work in different ways covers both aspects.
